google/resource_dns_record_set.go

```diff
@@ -286,6 +293,19 @@ func resourceDnsRecordSetUpdate(d *schema.ResourceData, meta interface{}) error
 	return resourceDnsRecordSetRead(d, meta)
 }
 
+func resourceDnsRecordSetImportState(d *schema.ResourceData, _ interface{}) ([]*schema.ResourceData, error) {
+	parts := strings.Split(d.Id(), "/")
+	if len(parts) != 3 {
+		return nil, fmt.Errorf("Invalid dns record specifier. Expecting {zone-name}/{record-name}/{record-type}")
```

Comment:
I think most people aren't going to know to add an extra . after the record name - might be a good place to tell them.

Reply:
Good idea. I also updated the docs to add a note about this.